pkg/genericapiserver/server/openapi/openapi.go

```diff
@@ -33,13 +33,15 @@ import (
 
 const (
 	OpenAPIVersion = "2.0"
+	ExtensionTags  = "x-kubernetes-tags"
```

Comment:
I've changed it to x-kubernetes-group-version-kind and also change the format of the content to specify which one is group, version, and kind.